gammapy/modeling/models/temporal.py

```diff
@@ -39,6 +39,12 @@ class TemporalModel(ModelBase):
 
     _type = "temporal"
 
+    def __init__(self, **kwargs):
+        scale = kwargs.pop("scale", "utc")
+        super().__init__(**kwargs)
+        if not hasattr(self, "scale"):
+            self.scale = scale
```

Comment:
It might be worth checking that scale is in `Time.SCALES`. Or at least check that the reference_time exists.

Reply:
A ConstantTemporalModel has no reference time